Templates/Swift/Sources/Request.swift

```diff
@@ -62,15 +62,14 @@ extension {{ options.name }}{% if tag %}.{{ options.tagPrefix }}{{ tag|upperCame
             public var {{ body.name}}: {{body.optionalType}}
             {% endif %}
 
-            public init({% if body %}{{ body.name}}: {{ body.optionalType }}{% if nonBodyParams %}, {% endif %}{% endif %}{% if nonBodyParams %}options: Options{% endif %}) {
+            public init({% if body %}jsonEncoder: JSONEncoder = JSONEncoder(), {{ body.name}}: {{ body.optionalType }}{% if nonBodyParams %}, {% endif %}{% endif %}{% if nonBodyParams %}options: Options{% endif %}) {
```

Comment:
Can we make this the last parameter of the initializer

Reply:
(First of all, I am sorry for the delay of my activity here 😓) I completely agree, (ideally) the order of mandatory arguments shouldn't change either you include the optional parameters or not.

I will also apply the change above mentioned.

Thanks for the feedback